src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -37,94 +37,55 @@
 import Vue, { PropType } from 'vue';
 import ReqCourse from '@/components/Requirements/ReqCourse.vue';
 import store from '@/store';
+import getCurrentSeason, { getCurrentYear } from '@/utilities';
 
-type CompletedSubReq = {
-  color: string;
-  transferCreditColor: string;
-  courseSubject: string;
-  courseNumber: string;
-  courseUniqueId: number;
-  isTransferCredit: boolean;
-  semesterType: FirestoreSemesterType;
-  semesterYear: number;
-};
+const transferCreditColor = 'DA4A4A'; // Arbitrary color for transfer credit
 
 export default Vue.extend({
   components: { ReqCourse },
   props: {
     subReqCourseId: { type: Number, required: true },
-    crsesTaken: { type: Array as PropType<readonly CourseTaken[]>, required: true },
-  },
-  data(): CompletedSubReq {
-    return {
-      semesterType: 'Fall', // 'Fall' is an arbitrary FirestoreSemesterType value that user will never see
-      semesterYear: 0,
-      color: '',
-      transferCreditColor: 'DA4A4A', // Arbitrary color for transfer credit
-      courseSubject: '',
-      courseNumber: '',
-      courseUniqueId: 0,
-      isTransferCredit: false,
-    };
-  },
-  mounted() {
-    const crseTaken = this.crsesTaken[0];
-    this.isTransferCredit = crseTaken.subject === 'AP' || crseTaken.subject === 'IB';
-    if (!this.isTransferCredit) {
-      this.setCourseAndSemesterForNonTransferCredits(crseTaken);
-    } else {
-      this.setCourseAndSemesterForTransferCredits(crseTaken);
-    }
+    courseTaken: { type: Object as PropType<CourseTaken>, required: true },
```

Comment:
we are always only use the first element in array, so we should just pass that element.